openassessment/xblock/grade_mixin.py

```diff
@@ -108,6 +108,7 @@ def render_grade_complete(self, workflow):
         has_submitted_feedback = False
 
         if "peer-assessment" in assessment_steps:
+            peer_api.get_score(submission_uuid, self.workflow_requirements()["peer"])
```

Comment:
Adding this line allows for the relevant peer assessments to be marked as "scored" so that they show up in the final grade report.

They are not actually used in calculating the final score, but it is necessary to differentiate which assessments would have been used to calculate the final score when getting the median for the final grade report.